modules/administration/pages/openscap.adoc

```diff
@@ -71,13 +71,38 @@ You are advised to use the [package]``scap-security-guide-redhat`` package you w
 [IMPORTANT]
 ====
 {suse} provides the [package]``scap-security-guide`` package for different openscap profiles.
-In the current version of [package]``scap-security-guide``, {suse} supports the following profiles:
-
-* DISA STIG profile for {sles} 12 and 15
-* PCI-DSS profile for {sles} 12 and 15
-* HIPAA profile for {sles} 12 and 15
-
-Other profiles, like the CIS profile, are community supplied and not officially supported by {suse}.
+In the current version of [package]``scap-security-guide``, {suse} supports the following profiles for {sles} 15:
```

Comment:
Keep the existing format and use the following list

- DISA STIG profile for SUSE Linux Enterprise Server 12 and 15
- ANSSI-BP-028 profile for SUSE Linux Enterprise Server 12 and 15
- PCI-DSS profile for SUSE Linux Enterprise Server 12 and 15
- HIPAA profile for SUSE Linux Enterprise Server 12 and 15
- CIS profile for SUSE Linux Enterprise Server 12 and 15
- Hardening for Public Cloud Image of SUSE Linux Enterprise Server for SAP Applications 15
- Public Cloud Hardening for SUSE Linux Enterprise 15